apps/storybook/src/TimePicker.stories.tsx

```diff
@@ -77,11 +84,171 @@ export const Basic: Story<TimePickerProps> = (args) => {
 Basic.args = {
   date: new Date(2021, 4, 11, 14, 55, 22),
   setFocusHour: true,
-  hourRenderer: (date: Date) =>
-    date.getHours().toLocaleString(undefined, { minimumIntegerDigits: 2 }),
-  minuteRenderer: (date: Date) =>
-    date.getMinutes().toLocaleString(undefined, { minimumIntegerDigits: 2 }),
-  secondRenderer: (date: Date) =>
-    date.getSeconds().toLocaleString(undefined, { minimumIntegerDigits: 2 }),
-  meridiemRenderer: (meridiem: MeridiemType) => meridiem,
+};
+
+export const CustomRenderers: Story<TimePickerProps> = (args) => {
```

Comment:
There is - Combined.

Reply:
We should probably document somewhere that virtualization is enabled by default. Maybe in the JSDocs.

Because in other components, we have to enable it explicitly. So users may think we don't support virtualization of this component because there is no `enableVirtualization` prop in `TimePickerProps`.

Reply:
In other components virtualization is determined by the data input of the user. Timepicker depends on that less so. I will include that info though.